ui/components/knowledge/base/_index.scss

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
+// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license
+
+/**
+ * @summary Initializes text input
+ *
+ * @name base
+ * @selector .nds-knowledge-block
+ * @restrict div
+ * @required
+ * @variant
+ */
+.nds-knowledge-block {
+  position: absolute;
+  top: 106px;
+  right: -8px;
+  width: 30%;
+  background: white;
+  bottom: 0;
+
+  .nds-right_panel {
+    height: 100%;
+  }
+
+  .nds-lookup.nds-knowledge-component {
+    height: 100%;
+
+    & > label.nds-form-element__label {
+      font-size: 14px;
```

Comment:
Can you switch the font-size to use rem instead of px? This will help with resizing for users who need larger font sizes for accessibility. See https://engageinteractive.co.uk/blog/em-vs-rem-vs-px for explanation.